client/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
   production: true,
+  RECAPTACHA_SITE_KEY: '6Ldbh14UAAAAAC_hGGi0xaHS5Om-Qln1UQV_lW-7', // For kfet-insa.fr domains
```

Comment:
Still not sure about this... we will have to find a way to do so without committing any key (except maybe the dev Key). It will be change at compilation time, so I think we should load the key from the environnement file (or any config file) and use it in the `update.sh` script

Reply:
We will talk about it tonight !